cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,491 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 4.1.0 <http://docs.cask.co/cdap/4.1.0/index.html>`__
+=============================================================
+
+New Features
+------------
+
+- :cask-issue:`CDAP-4556` - CDAP now uses environment variables in the ``spark-env.sh`` and
```

Comment:
I think within each section, the ordering should stay that way (because if someone is looking for an issue, it makes it harder to find). But we can create other sections with different ordering, such as a "highlights" section.